packages/xod-client-electron/src/view/containers/App.jsx

```diff
@@ -575,7 +575,19 @@ class App extends client.App {
         items.cut,
         items.copy,
         items.paste,
-        items.selectall,
+        onClick(items.selectall, () => {
```

Comment:
Too complex function for the purely structural code span. Suggest extracting it to a method